sunpy/physics/differential_rotation.py

```diff
@@ -25,12 +26,14 @@
 from sunpy.sun.constants import sidereal_rotation_rate
 from sunpy.time import parse_time
 from sunpy.util import expand_list
+from sunpy.util.decorators import deprecated
 from sunpy.util.exceptions import warn_user
 
 __all__ = ['diff_rot', 'solar_rotate_coordinate', 'differential_rotate']
 
 
 @u.quantity_input
+@deprecated(since="6.0", alternative="sunpy.sun.models.diff_rot")
 def diff_rot(duration: u.s, latitude: u.deg, rot_type='howard', frame_time='sidereal'):
```

Comment:
We want to avoid code duplication so if you could import the new function inside this function and call it and return the result.

Reply:
Sure. I will do it.